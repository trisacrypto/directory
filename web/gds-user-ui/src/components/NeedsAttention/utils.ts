export const enum AttentionAction {
  'VERIFY_EMAILS' = 'VERIFY_EMAILS',
  'VERIFY_PHONES' = 'VERIFY_PHONES',
  'VERIFY_ADDRESSES' = 'VERIFY_ADDRESSES',
  'VERIFY_IDENTITIES' = 'VERIFY_IDENTITIES',
  'VERIFY_DOCUMENTS' = 'VERIFY_DOCUMENTS',
  'VERIFY_ALL' = 'VERIFY_ALL',
  'VERIFY_NONE' = 'VERIFY_NONE'
}
