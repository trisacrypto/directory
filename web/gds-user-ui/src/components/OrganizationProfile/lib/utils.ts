export const StatusCheck = (status: string) => {
  switch (status) {
    case 'NO_VERIFICATION':
      return 'Not Verified';
    case 'PENDING_REVIEW':
      return 'Pending Review';
    case 'VERIFIED':
      return 'Verified';
    case 'REJECTED':
      return 'Rejected';
    default:
      return 'Pending Registration';
  }
};
