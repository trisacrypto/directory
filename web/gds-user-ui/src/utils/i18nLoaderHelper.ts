import { i18n } from '@lingui/core';
import { en, fr, de, zh, ja, ru } from 'make-plural/plurals';
import { Locales } from 'types/type';

export const DEFAULT_LOCALE: Locales = 'en';

export async function dynamicActivate(locale: string) {
  const { messages } = await import(`../locales/${locale}/messages`);

  i18n.load(locale, messages);
  i18n.loadLocaleData({
    en: { plurals: en },
    de: { plurals: de },
    fr: { plurals: fr },
    zh: { plurals: zh },
    ja: { plurals: ja }
  });
  i18n.activate(locale);
}
