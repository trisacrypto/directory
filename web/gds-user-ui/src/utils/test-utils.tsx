import { ChakraProvider } from '@chakra-ui/react';
import { render as rtlRender, RenderOptions as RTLRenderOptions } from '@testing-library/react';
import React from 'react';
import { FormProvider, useForm } from 'react-hook-form';
import { Provider } from 'react-redux';
import customTheme from './theme';
import rootReducer from 'application/store/rootReducer';
import { configureStore, Store } from '@reduxjs/toolkit';

type RenderOptions = {
  rhfDefaultValues?: { [x: string]: any };
  store?: Store;
  preloadedState?: StoreState;
  renderOptions?: Omit<RTLRenderOptions, 'wrapper'>;
};

type StoreState = {
  [x: string]: any;
};

export function render(
  ui: React.ReactElement<any, string | React.JSXElementConstructor<any>>,
  {
    rhfDefaultValues,
    preloadedState,
    store = configureStore({ reducer: rootReducer, preloadedState }),
    ...renderOptions
  }: RenderOptions = {}
) {
  function Wrapper({ children }: { children: React.ReactNode }) {
    const methods = useForm({ defaultValues: rhfDefaultValues });
    return (
      <Provider store={store}>
        <ChakraProvider theme={customTheme}>
          <FormProvider {...methods}>{children}</FormProvider>
        </ChakraProvider>
      </Provider>
    );
  }
  return rtlRender(ui, { wrapper: Wrapper, ...renderOptions });
}

export * from '@testing-library/react';
