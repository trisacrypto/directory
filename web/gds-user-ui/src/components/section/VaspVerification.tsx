import React from 'react';
import {
  Stack,
  Container,
  Box,
  Flex,
  Text,
  Link,
  chakra,
  FlexProps,
  StyleProps,
  useColorModeValue,
  UnorderedList,
  ListItem,
  Button
} from '@chakra-ui/react';

import { colors } from '../../utils/theme';

type Props = StyleProps &
  FlexProps & {
    children: React.ReactNode;
    title?: string;
  };

interface ILineProps {
  children?: React.ReactNode;
  title?: string;
}

const Line: React.FC<Props> = ({ children, title, ...rest }: any) => {
  return (
    <Flex>
      <Flex shrink={0}>
        <Flex rounded="md" bg={useColorModeValue('brand.500', '')} color="white"></Flex>
      </Flex>
      <Box ml={4}>
        <chakra.dt fontSize="lg" fontWeight="medium" lineHeight="6" {...rest}>
          {title}
        </chakra.dt>
        <chakra.dd mt={2}>{children}</chakra.dd>
      </Box>
    </Flex>
  );
};
export default function VaspVerification() {
  return (
    <Flex color={'black'} fontFamily={'Open Sans'} fontSize={'xl'} px={40}>
      <Stack>
        <Stack flex={1} justify={{ lg: 'center' }} py={{ base: 4, md: 20 }}>
          <Box my={{ base: 4 }} color="black">
            <Text fontFamily={'heading'} fontWeight={700} fontSize={'xl'}>
              TRISA members must complete a comprehensive multi-part verification form and due
              diligence process. Once verified, TRISA will issue TestNet and MainNet certificates
              for secure Travel Rule compliance.
            </Text>
          </Box>
          <Box bg={'gray.100'} p={5}>
            <Text fontSize={'xl'} color={'black'}>
              TRISA is the only global, open source, peer-to-peer and secure Travel Rule network.
              Become a TRISA-certified VASP today. Learn how TRISA works.
            </Text>
          </Box>
          <Box mt={20} pt={10}>
            <Stack
              display={{ md: 'grid' }}
              gridTemplateColumns={{ md: 'repeat(2,1fr)' }}
              color={'black'}
              gridColumnGap={{ md: 20, lg: 80 }}
              gridRowGap={{ md: 10 }}>
              <>
                <Line title="Sections & Details" fontWeight={'bold'}>
                  {''}
                </Line>
                <Line title="Who to Ask" fontWeight={'bold'}>
                  {''}
                </Line>
              </>

              <Line title="1 Basic Details" fontWeight={'bold'}>
                Information about the VASP such as website, incorporation date, business and VASP
                category.
              </Line>

              <Line>Business or Compliance Office</Line>

              <Line title="2 Legal Person" fontWeight={'bold'}>
                Information that identifies your organization as a Legal Person. This section
                represents the IVMS 101 data structure for legal persons and is strongly suggested
                for use as KYC information exchanged in TRISA transfers.
              </Line>
              <Line>Business or Compliance Office</Line>

              <Line title="3 Contacts" fontWeight={'bold'}>
                Contact information for representatives of your organization. Contacts include
                Technical, Legal/Compliance, Administrative, and Billing persons.
              </Line>
              <Line>Business or Compliance Office</Line>

              <Line title="4 TRISA Implementation" fontWeight={'bold'}>
                Technical information about your endpoint for certificate issuance. Each VASP is
                required to establish a TRISA endpoint for inter-VASP communication.
              </Line>
              <Line>Technical Officer</Line>
              <Line title="5 TRIXO Questionnaire" fontWeight={'bold'}>
                information to ensure that required compliance information exchanges are conducted
                correctly and safely. This includes information about jurisdiction and national
                regulator, CDD and Travel Rule policies, and data protection policies.
              </Line>
              <Line>Compliance Officer</Line>
              <Stack mt={20} bg={'gray.100'} py={5}>
                <Line title="Final Confirmation" fontWeight={'bold'}>
                  Upon submission, a member of TRISAâ€™s verification team will review the form and
                  conduct a final due diligence phone call for physical verfication. Once due
                  diligence is complete, TRISA will issue certificates to the VASP.
                </Line>
              </Stack>
              <Stack mt={20} bg={'gray.100'} py={5}>
                <Line title="Need to Learn More?" fontWeight={'bold'}>
                  <UnorderedList>
                    <ListItem>
                      <Link>How TRISA Works</Link>
                    </ListItem>
                    <ListItem>
                      <Link>What is IVMS101?</Link>
                    </ListItem>
                  </UnorderedList>
                </Line>
              </Stack>
            </Stack>
          </Box>
          <Stack direction={['column', 'row']} pt={20} mx={10}>
            <Box>
              <Button
                bg={colors.system.blue}
                color={'white'}
                _hover={{
                  bg: '#10aaed'
                }}
                _focus={{
                  borderColor: 'transparent'
                }}>
                Download PDF
              </Button>
            </Box>
            <Box>
              <Button
                bg={colors.system.blue}
                color={'white'}
                _hover={{
                  bg: '#10aaed'
                }}
                _focus={{
                  borderColor: 'transparent'
                }}>
                Back to Getting Started
              </Button>
            </Box>
          </Stack>
        </Stack>
      </Stack>
    </Flex>
  );
}
