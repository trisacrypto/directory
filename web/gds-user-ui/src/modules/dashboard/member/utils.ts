import { DirectoryType, VaspDirectoryEnum, DirectoryTypeEnum } from './memberType';
import { convertToCVS, downloadCSV } from 'utils/utils';
import { t } from '@lingui/macro';

export const memberTableHeader = [
  {
    key: 'name',
    label: t`Member Name`
  },
  {
    key: 'first_listed',
    label: t`Joined`
  },
  {
    key: 'last_updated',
    label: t`Last Updated`
  },
  {
    key: 'registered_directory',
    label: t`Network`
  },
  {
    key: 'status',
    label: t`Status`
  }
];
export const getVaspDirectory = (dir: DirectoryType) => {
  return dir === DirectoryTypeEnum.TESTNET ? VaspDirectoryEnum.TESTNET : VaspDirectoryEnum.MAINNET;
};

export const getVaspNetwork = (dir: any) => {
  switch (dir) {
    case (VaspDirectoryEnum.TESTNET, VaspDirectoryEnum.TESTNET_DEV):
      return 'TestNet';
    case (VaspDirectoryEnum.MAINNET, VaspDirectoryEnum.MAINNET_DEV):
      return 'MainNet';
    default:
      return 'MainNet';
  }
};

export const getVaspStatus = (status: number) => {
  switch (status) {
    case 1:
      return t`NO VERIFICATION`;
    case 2:
      return t`SUBMITTED`;
    case 3:
      return t`EMAIL VERIFIED`;
    case 4:
      return t`PENDING REVIEW`;
    case 5:
      return t`REVIEWED`;
    case 6:
      return t`VERIFIED`;
    case 7:
      return t`REJECTED`;
    case 8:
      return t`APPEALED`;
    case 9:
      return t`ERRORED`;
    default:
      return t`NO VERIFICATION`;
  }
};

export const downloadMembers2CVS = (member: any) => {
  const m = member.map((item: any) => {
    return {
      ...item,
      status: getVaspStatus(item.status),
      registered_directory: getVaspNetwork(item.registered_directory)
    };
  });
  const memberCsv = convertToCVS(m, memberTableHeader as ITableHeader[]);
  downloadCSV(memberCsv, 'members');
};

