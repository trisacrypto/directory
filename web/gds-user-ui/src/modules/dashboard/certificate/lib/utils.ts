import { basicDetailsValidationSchema } from 'modules/dashboard/certificate/lib/basicDetailsValidationSchema';
import { legalPersonValidationSchemam } from 'modules/dashboard/certificate/lib/legalPersonValidationSchema';
import { contactsValidationSchema } from 'modules/dashboard/certificate/lib/contactsValidationSchema';
import { trisaImplementationValidationSchema } from 'modules/dashboard/certificate/lib/trisaImplementationValidationSchema';
import { trixoQuestionnaireValidationSchema } from 'modules/dashboard/certificate/lib/trixoQuestionnaireValidationSchema';

export const isBasicDetailsCompleted = async (data: any) => {
  try {
    const r = await basicDetailsValidationSchema.validate(data);
    console.log(r);
    return true;
  } catch (err) {
    return false;
  }
};

export const isLegalPersonCompleted = async (data: any) => {
  try {
    const r = await legalPersonValidationSchemam.validate(data);
    console.log(r);
    return true;
  } catch (err) {
    console.log('legal person validation failed', err);
    return false;
  }
};

export const isContactsCompleted = async (data: any) => {
  try {
    const r = await contactsValidationSchema.validate(data);
    console.log(r);
    return true;
  } catch (err) {
    console.log('contacts validation failed', err);
    return false;
  }
};

export const isTrisaImplementationCompleted = async (data: any) => {
  try {
    await trisaImplementationValidationSchema.validate(data);
    return true;
  } catch (err) {
    return false;
  }
};

export const isTrixoQuestionnaireCompleted = async (data: any) => {
  try {
    await trixoQuestionnaireValidationSchema.validate(data);
    return true;
  } catch (err) {
    return false;
  }
};
