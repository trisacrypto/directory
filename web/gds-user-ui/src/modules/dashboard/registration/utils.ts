import { getDefaultValue } from 'components/BasicDetailsForm/validation';
import { getRegistrationDefaultValues } from 'modules/dashboard/certificate/lib';
import {
  postRegistrationData,
  getRegistrationData,
  getSubmissionStatus
} from 'modules/dashboard/registration/service';
import { handleError } from 'utils/utils';

export const postRegistrationValue = (data: any) => {
  console.log('[postRegistrationValue]', data);
  return new Promise((resolve, reject) => {
    postRegistrationData(data)
      .then((res) => {
        console.log('[postRegistrationData]', res);
        if (res.status === 204) {
          resolve(res);
        } else {
          reject(res);
        }
      })
      .catch((err) => {
        console.log('[postRegistrationData]', err);
        reject(err);
      });
  });
};
export const getRegistrationDefaultValue = async () => {
  try {
    const regData = await getRegistrationData();
    if (regData.status === 200 && Object.keys(regData.data).length > 0) {
      return regData.data;
    }
    const defaultValue: any = localStorage.getItem('certificateForm');
    if (defaultValue) {
      const val = JSON.parse(defaultValue);
      const postData = await postRegistrationData(val);
      if (postData.status === 204) {
        const getData = await getRegistrationData();
        localStorage.removeItem('certificateForm');
        return getData.data;
      }
    }
    const v = getRegistrationDefaultValues();
    await postRegistrationValue(v);
    return getRegistrationDefaultValues();
  } catch (err: any) {
    handleError(err, 'failed to get registration data');
  }
};

export const setRegistrationDefaultValue = () => {
  console.log('[setRegistrationDefaultValue]');
  const defaultValue: any = getRegistrationDefaultValues();
  return new Promise((resolve, reject) => {
    postRegistrationData(defaultValue)
      .then((res) => {
        // console.log('[default postRegistration value]', res);
        if (res.status === 204) {
          resolve(res);
        } else {
          reject(res);
        }
      })
      .catch((err) => {
        handleError(err, 'failed to post registration value');
        reject(err);
      });
  });
};

// get registration data from backend and download the file
export const downloadRegistrationData = async () => {
  try {
    const regData = await getRegistrationData();
    if (regData.status === 200 && Object.keys(regData.data).length > 0) {
      const blob = new Blob([JSON.stringify(regData.data)], { type: 'application/json' });
      const url = window.URL.createObjectURL(blob);
      const link = document.createElement('a');
      link.href = url;
      link.setAttribute('download', 'registration.json');
      document.body.appendChild(link);
      link.click();
    }
  } catch (err: any) {
    handleError(err, 'failed to get registration data');
  }
};

// load default stepper

export const getDefaultStepper = async () => {
  try {
    const [regData, regStatus] = await Promise.all([getRegistrationData(), getSubmissionStatus()]);
    console.log('[regStatus]', regStatus.data);
    if (regData.status === 200 && Object.keys(regData.data).length > 0) {
      return {
        currentStep: regData.data.state.current,
        steps: regData.data.state.steps,
        lastStep: null,
        hasReachSubmitStep: regData.data.state.ready_to_submit,
        testnetSubmitted: regStatus?.data?.testnetSubmitted || false,
        mainnetSubmitted: regStatus?.data?.mainnetSubmitted || false
      };
    }
    const defaultValue: any = {
      current: 1,
      steps: [
        {
          key: 1,
          status: 'progress'
        }
      ]
    };
    // update registrations state object
    const postData = await postRegistrationData({ state: defaultValue });
    if (postData.status === 204) {
      const getData = await getRegistrationData();
      return {
        currentStep: getData.data.state.current,
        steps: getData.data.state.steps,
        lastStep: null,
        hasReachSubmitStep: false,
        testnetSubmitted: false,
        mainnetSubmitted: false
      };
    }
  } catch (err: any) {
    handleError(err, 'failed to get stepper data');
  }
};

// // load default stepper without async call
// export const loadDefaultStepperSync = () => {
//   return Promise.resolve({
//     currentStep: 1,
//     steps: [
//       {
//         key: 1,
//         status: 'progress'
//       }
//     ],
//     lastStep: null,
//     hasReachSubmitStep: false
//   });
// };

// set stepper data
export const getRegistrationAndStepperData = async () => {
  try {
    const [regData, regStatus] = await Promise.all([
      getRegistrationDefaultValue(),
      getSubmissionStatus()
    ]);
    if (regData) {
      const response: any = {
        registrationData: regData,
        stepperData: {
          currentStep: regData?.state?.current || 1,
          steps: regData?.state?.steps || [{ key: 1, status: 'progress' }],
          lastStep: null,
          hasReachSubmitStep: regData?.state?.ready_to_submit || false,
          testnetSubmitted: !!regStatus?.data?.testnet_submitted,
          mainnetSubmitted: !!regStatus?.data?.mainnet_submitted
        }
      };
      return response;
    }
  } catch (err: any) {
    handleError(err, 'failed to get stepper data');
  }
};
