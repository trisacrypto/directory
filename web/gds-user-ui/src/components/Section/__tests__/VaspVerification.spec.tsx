import { dynamicActivate } from 'utils/i18nLoaderHelper';
import { act, render, screen } from 'utils/test-utils';
import VaspVerification from '../VaspVerification';

describe('<Line />', () => {
  beforeAll(() => {
    act(() => {
      dynamicActivate('en');
    });
  });

  it('should match snapshot', () => {
    const { container } = render(<VaspVerification />);

    expect(container).toMatchSnapshot();
  });
});
