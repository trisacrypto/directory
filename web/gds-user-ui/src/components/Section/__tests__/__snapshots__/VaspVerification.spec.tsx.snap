// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Line /> should match snapshot 1`] = `
<div>
  <div
    class="css-1nkacyh"
  >
    <div
      class="chakra-container css-1a3ltpp"
    >
      <div
        class="chakra-stack css-12m5edu"
      >
        <div
          class="css-1y81stp"
        >
          <p
            class="chakra-text css-1t2l67w"
          >
            TRISA members must complete a comprehensive multi-part verification form and due diligence process. Once verified, TRISA will issue TestNet and MainNet certificates for secure Travel Rule compliance.
          </p>
        </div>
        <div
          class="css-7egbnq"
        >
          <p
            class="chakra-text css-1wbbpf0"
          >
            TRISA’s verification form includes five sections and may require information from several parties in your organization.
          </p>
        </div>
        <div
          class="css-cspnrf"
        >
          <div
            class="css-1ipaw1x"
          >
            <div
              class="css-rklm6r"
            >
              <dt
                class="css-1ejj06g"
              >
                Sections & Details
              </dt>
              <dd
                class="css-a3f1ip"
              />
            </div>
            <div
              class="css-tuh9u2"
            >
              <dt
                class="css-1ejj06g"
              >
                Who to Ask
              </dt>
              <dd
                class="css-a3f1ip"
              />
            </div>
            <div
              class="css-rklm6r"
            >
              <dt
                class="css-1ejj06g"
              >
                1 Basic Details
              </dt>
              <dd
                class="css-a3f1ip"
              >
                Information about the VASP such as website, incorporation date, business and VASP category.
              </dd>
            </div>
            <div
              class="css-tuh9u2"
            >
              <dt
                class="css-1ub76a4"
              />
              <dd
                class="css-a3f1ip"
                data-content="Who to ask"
              >
                Business or Compliance Office
              </dd>
            </div>
            <div
              class="css-rklm6r"
            >
              <dt
                class="css-1ejj06g"
              >
                2 Legal Person
              </dt>
              <dd
                class="css-a3f1ip"
              >
                Information that identifies your organization as a Legal Person. This section represents the IVMS 101 data structure for legal persons and is strongly suggested for use as KYC information exchanged in TRISA transfers.
              </dd>
            </div>
            <div
              class="css-tuh9u2"
            >
              <dt
                class="css-1ub76a4"
              />
              <dd
                class="css-a3f1ip"
                data-content="Who to ask"
              >
                Business or Compliance Office
              </dd>
            </div>
            <div
              class="css-rklm6r"
            >
              <dt
                class="css-1ejj06g"
              >
                3 Contacts
              </dt>
              <dd
                class="css-a3f1ip"
              >
                Contact information for representatives of your organization. Contacts include Technical, Legal/Compliance, Administrative, and Billing persons.
              </dd>
            </div>
            <div
              class="css-tuh9u2"
            >
              <dt
                class="css-1ub76a4"
              />
              <dd
                class="css-a3f1ip"
                data-content="Who to ask"
              >
                Business or Compliance Office
              </dd>
            </div>
            <div
              class="css-rklm6r"
            >
              <dt
                class="css-1ejj06g"
              >
                4 TRISA Implementation
              </dt>
              <dd
                class="css-a3f1ip"
              >
                Technical information about your endpoint for certificate issuance. Each VASP is required to establish a TRISA endpoint for inter-VASP communication.
              </dd>
            </div>
            <div
              class="css-tuh9u2"
            >
              <dt
                class="css-1ub76a4"
              />
              <dd
                class="css-a3f1ip"
                data-content="Who to ask"
              >
                Technical Officer
              </dd>
            </div>
            <div
              class="css-rklm6r"
            >
              <dt
                class="css-1ejj06g"
              >
                5 TRIXO Questionnaire
              </dt>
              <dd
                class="css-a3f1ip"
              >
                Information to ensure that required compliance information exchanges are conducted correctly and safely. This includes information about jurisdiction and national regulator, Customer Due Diligence(CDD) and Travel Rule policies, and data protection policies.
              </dd>
            </div>
            <div
              class="css-tuh9u2"
            >
              <dt
                class="css-1ub76a4"
              />
              <dd
                class="css-a3f1ip"
                data-content="Who to ask"
              >
                Compliance Officer
              </dd>
            </div>
          </div>
          <div
            class="css-gus2rx"
          >
            <div
              class="css-11j9xqf"
            >
              <dt
                class="css-kn4i7c"
              >
                Final Confirmation
              </dt>
              <dd
                class="css-1n004p3"
              >
                For MainNet certificate requests, a member of TRISA’s verification team will review your submission and conduct a final due diligence phone call for physical verification. When physical verification is complete, TRISA will issue MainNet certificates. Requests for TestNet certificates do not require physical verification.
              </dd>
            </div>
            <div
              class="css-gwmn0b"
            >
              <dt
                class="css-kn4i7c"
              >
                Need to Learn More?
              </dt>
              <dd
                class="css-rltemf"
              >
                <ul
                  class="css-xuskyr"
                  role="list"
                >
                  <li
                    class="css-14d0z92"
                  >
                    <a
                      class="chakra-link css-f4h6uy"
                      href="https://trisa.io/getting-started-with-trisa/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      Learn How TRISA Works
                    </a>
                  </li>
                  <li
                    class="css-14d0z92"
                  >
                    <a
                      class="chakra-link css-f4h6uy"
                      href="https://intervasp.org/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      What is IVMS101?
                    </a>
                  </li>
                </ul>
              </dd>
            </div>
          </div>
        </div>
        <div
          class="chakra-stack css-15av81"
        >
          <div
            class="css-1tms7qa"
          >
            <a
              class="chakra-button css-1n03jlk"
              href="/certificate/registration"
            >
              Start Registration Process
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
